### ISSUE TITLE
Problem on android 5.0

### ISSUE THREAD

**Issue by edimilsonramos** [cowbell/cordova-plugin-geofence#34]
Running on android 5.0 the function initialize unresponsive success or error and consequently does not work for me.
What could be wrong?
A small and simple part of my code:

function onDeviceReady(){
     window.geofence.receiveTransition = function (geofences) {
        geofences.forEach(function (geo) {
            console.log('Geofence transition detected', geo);
            $(document).find('.status').html('Geofence transition detected');
            alert('Geofence transition detected' + geo.id);
        });
    };

    try {
        window.geofence.initialize(onSucessInit, onErrorInit);
    }
    catch(err) {
        alert('Erro try ---' + err.message);
    }
    startAPP();

}

function onSucessInit(){
    alert('OK init');
}

function onErrorInit(){
    alert('Error init');
}

**Comment by tsubik** [comment]
Hey @edimilsonramos. I am sorry for not responding so long. Do you still have some problems with this plugin? `initialize` method for android does nothing so far and I need to remove it maybe.

**Comment by andiCR** [comment]
Hello @tsubik , you should call the callbackContext.success("") from the initialize action on the GeofencePlugin execute function, since this is relevant to other platforms. For iOS i need some code to be executed upon plugin initialization. I don't want to have separate code for android and for iOS, and the plugin should behave exactly the same on both platforms, wether it does anything on Android or not. 

In my code I do something like this:

      window.geofence.initialize(function(){

        window.geofence.onTransitionReceived = geofenceManager.onTransitionReceived;

        geofenceManager.log("Geofence initialized.");

        // Get current geofences and remove events that have already passed
        window.geofence.getWatched().then(function (geofencesJson) {
            ...
        });

So basically, please add the following to GeofencePlugin.java

    @Override
    public boolean execute(String action, JSONArray args,
            CallbackContext callbackContext) throws JSONException {
        ...
        } else if (action.equals("initialize")) {
            callbackContext.success("");
        }
        ...
        return true;

    }

**Comment by tsubik** [comment]
Fixed via #55
